tensorflow/core/api_def/base_api/api_def_Bitcast.pbtxt

```diff
@@ -12,6 +12,33 @@ If `T` is smaller than `type`, the operator requires that the rightmost
 dimension be equal to sizeof(`type`)/sizeof(`T`). The shape then goes from
 [..., sizeof(`type`)/sizeof(`T`)] to [...].
 
+tf.bitcast() is used when you have to cast to an unsigned dtype (e.g. tf.uint8)
+where tf.cast() will not work effectively.
+
+```
+python
+code here
+```
+For example,
+  tf.bitcast(tf.cast(my_tensor, tf.int8), tf.uint8)
+
+Other example,
```

Comment:
I think the key thing you are trying to show in this second example is how bitcast will treat casting where the input and output dtype have a different number of bits? In that case I think it might be simplest to simply use one line:

```python
>>> tf.bitcast(tf.constant(0xffffffff, dtype=tf.uint32), tf.uint8)
<tf.Tensor: ... shape=(4,), dtype=uint8, numpy=array([255, 255, 255, 255], dtype=uint8)>
```

Reply:
Ok i will replace my first example with  this line.